### PR TITLE
DCKUBE-289 resource configuration recommendation

### PR DESCRIPTION
**Pull request by nghazali**
- provided a link to recommended data center infrastructure in AWS
- moved the horizontal scaling section from INSTALL.md to scaling document

### CODE REVIEW

File: docs/resource_management/RESOURCE_SCALING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+The Helm charts provision one `StatefulSet` by default. In order to horizontally scale up or down the cluster `kubectl scale` 
+can be used at runtime to provision a multi-node Data Center cluster, with no further configuration required (although 
+note that the Ingress must support cookie-based session affinity in order for the 
+products to work correctly in a multi-node configuration). Here is the syntax for scaling up/down the Data Center cluster:
+```
+kubectl scale statefulsets <statefulsetset-name> --replicas=n
+```
+For details on the `cpu` and `memory` requirements of the product `StatfulSets` see [Resource requests and limits](#resource-requests-and-limits) 
+
+##Vertical Scaling
```
Comment (nanux) [inline docs/resource_management/RESOURCE_SCALING.md:24]:
```suggestion
## Vertical Scaling
```

File: docs/product_upgrades/ROLLING_UPGRADE.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ In this strategy users select the pod to upgrade by deleting it, and Kubernetes
 ```  
 
 ## BitBucket Rolling Upgrade
```
Comment (nanux) [inline docs/product_upgrades/ROLLING_UPGRADE.md:26]:
```suggestion
## Bitbucket Rolling Upgrade
```

File: docs/resource_management/RESOURCE_SCALING.md (hunk 1 above)
Comment (nghazali) [reply docs/resource_management/RESOURCE_SCALING.md]: done

File: docs/product_upgrades/ROLLING_UPGRADE.md (hunk 2 above)
Comment (nghazali) [reply docs/product_upgrades/ROLLING_UPGRADE.md]: done

**Pull request closed without merging**